uniffi_bindgen/src/bindings/swift/templates/CallbackInterfaceTemplate.swift

```diff
@@ -0,0 +1,134 @@
+{% let type_name = cbi.name()|class_name_swift %}
+public protocol {{ type_name }} : AnyObject {
+    {% for meth in cbi.methods() -%}
+    func {{ meth.name()|fn_name_swift }}({% call swift::arg_list_protocol(meth) %}) {% call swift::throws(meth) -%}
+    {%- match meth.return_type() -%}
+    {%- when Some with (return_type) %} -> {{ return_type|type_swift -}}
+    {%- else -%}
+    {%- endmatch %}
+    {% endfor %}
+}
+
+{% let canonical_type_name = cbi.type_().canonical_name()|class_name_swift %}
+{% let callback_internals = format!("{}Internals", canonical_type_name) -%}
+{% let callback_interface_impl = format!("{}FFI", canonical_type_name) -%}
```

Comment:
The `callback_interface_impl` is now a variable identifier rather than another class/protocol:

```suggestion
{% let callback_interface_impl = format!("{}", canonical_type_name)|var_name_swift -%}
```

Reply:
I tried this, but didn't manage to get it working, could you post the full change required?

Reply:
👍